src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>My Account</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="firstName"
+    ></v-text-field>
```

Comment:
instead of <tag {data} ></tag> you can do a single <tag {data}/>

Reply:
ok, corrected on 
[a71e249](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/a71e249f6c074952cf0061a60c288e4f9804268d)

Reply:
ok but it depends on the tag some tags require a `<tag></tag>`